public/store/css/main.css

```diff
@@ -132,9 +132,10 @@ input[type="radio"]:checked+label:after {
 }
 
 .styled-checkbox:checked+label:after {
-	content: "✔";
-	top: 8px;
-	left: 12px;
+	font-family: FontAwesome, 'Helvetica Neue', Helvetica, Arial, sans-serif;
+	content: "\f00c";
```

Comment:
Does that code work in the other fonts?

Reply:
Otherwise I think remove them as fallbacks or add as a more standard unicode char? https://www.fileformat.info/info/unicode/char/2714/index.htm

Reply:
We did use that, it was the issue - the heavy check mark was showing as a weird green tick in edge, seemingly regardless of the font-family setting. 

Forcing it to use FontAwesome's icon is both more in keeping with the other icons we use and consistent across browsers. If FontAwesome fails to load lots of other icons will be broken.

Reply:
I've removed the excess, pointless members. PhpStorm seems to think we need a default standard font-family too, hence sans-serif, to fall back on, but i don't expect it to ever be used.